userspace/falco/configuration.h

```diff
@@ -100,6 +100,7 @@ class falco_configuration {
 		std::set<std::string> m_tags;
 		std::string m_rule;
 		std::string m_format;
+		bool m_suggested_output = false;
```

Comment:
When missing, default value is false.

Reply:
Is there any particular reason why it is enabled by default in the config but disabled if missing? 

Not saying this is wrong, I'm just wondering why.

Reply:
That's because i want it to be effective only when explicitly enabled; i don't want it to have any effect if you specify eg:
```
append_output:
   - match:
       source: syscall
     extra_output: "on CPU %evt.cpu"
```
Ie: in this example, i only want `on CPU %evt.cpu` to be appended to output, not the plugin suggested fields, unless `suggested_output: true` is also set.

But i am also open for more opinions.